src/components/core/Category.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Category = ({ name, onClick }) => {
```

Comment:
import PropTypes from "prop-types";
use **PropTypes** to ensure the correct datatype is passed for each prop

Reply:
Resolved ✅